msteams-platform/sbs-inmeeting-document-signing.yml

```diff
@@ -0,0 +1,473 @@
+### YamlMime:Tutorial
+title: Build in-meeting document signing
+metadata:
+  title: Build in-meeting document signing app
+  description: With this learning module, learn how to build in-meeting document signing app in Teams.
+  audience: Developer
+  level: Beginner
+  ms.date: 27/07/2021
+  ms.topic: interactive-tutorial
+  nextTutorialHref: apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md
+  nextTutorialTitle: Read more to build apps for Teams meeting stage
+  ms.custom: mvc
+  ms.localizationpriority: high
+items:
+- durationInMinutes: 1
+  content: |
+    Meeting signing is a Microsoft Teams sample app that enables users to sign documents in real time scenarios. It provides enhanced experience by allowing the users to review and sign documents in a single session during meetings.
+    This sample lets users electronically sign documents with their current tenant identity. The users can sign the following documents during meetings:
+    * Purchase agreements.
+    * Purchase orders.
+    
+    The template is also applicable for other scenarios, such as for reviewing and approving code samples and collaborative documents.
+    The users can customize to incorporate other types of signatures: 
+    * Handwritten signatures.
+    * Certificate-based signatures.
+
+    This step-by-step guide helps you to build in-meeting document signing app. You'll see the following output after you've completed this guide:
+
+      :::image type="content" source="~/assets/images/sbs-inmeeting-doc-signing/final-output.png" alt-text="Final Output":::
+
+- title: Prerequisites
+  durationInMinutes: 1
+  content: |
+    Ensure you install the following tools for the development environment:  
+
+    * [Latest version of ngrok](https://ngrok.com/download) or any equivalent tunneling solution.
+
+      > [!NOTE]
+      > After downloading ngrok, sign up and install [authtoken](https://ngrok.com/download).
+
+    * [Latest version of Visual Studio](https://visualstudio.microsoft.com/downloads/).
+    * [Latest version of.NET Core SDK](https://dotnet.microsoft.com/download).
+
+    Ensure you have created the following accounts to set up your development environment: 
+
+    * [Microsoft Teams](https://teams.microsoft.com/) account.
+
+      > [!TIP]
+      > Ensure the Microsoft Teams account isn't a guest account.
+
+    * [Microsoft 365 developer account](https://developer.microsoft.com/en-us/microsoft-365/dev-program). If you don't have that, you need have access to a Teams account with the appropriate permissions to install an app.
+
+    ## Set up your Teams development tenant
+
+       A **tenant** is like a space or a container where you chat, share files, and run meetings for your organization in Teams. You can also sideload and test the app.
+
+    ### Check for sideloading option
```

Comment:
This reads weirdly. Perhaps something like "Check sideloading of apps is enabled" makes more sense

Reply:
Thank you for your suggestion. This is a common heading that is used for several sections. As sideloading needs to be explained, the heading has been kept generic. A new user may not know what sideloading an app might mean.